src/coffee/directives/search-box.coffee

```diff
@@ -42,7 +42,7 @@ angular.module('uiGmapgoogle-maps')
             $templateCache.put 'uigmap-searchbox-default.tpl.html', '<input type="text">'
             scope.template = 'uigmap-searchbox-default.tpl.html'
           $http.get(scope.template, { cache: $templateCache })
-            .success (template) =>
+            .then (template) =>
```

Comment:
I think you'll need `.then (template) => template.data` or `.then ({data) =>` here as well.